examples.md

```diff
@@ -550,3 +551,17 @@ whenever possible:
     restore-keys: |
       ${{ runner.os }}-spm-
 ```
+
+## * - Bazel
+
+Bazel cache has a good handle to check if cached content should be rebuild or not based on its inputs like a hash(command + files). So using the latest cache of the branch is enough, no need to suffix with `hashFiles('**/...')`.
+
+```yaml
+- name: Cache Bazel
+  uses: actions/cache@v2
+  with:
+    path: |
+      ~/.cache/bazelisk
+      ~/.cache/bazel
```

Comment:
This works for Linux, but you're missing macOS (`/private/var/tmp/_bazel_runner/`) and Windows (???) paths.

Reply:
Re. Windows, https://bazel.build/remote/output-directories says _"on Windows it defaults to %HOME% if set, else %USERPROFILE% if set, else the result of calling SHGetKnownFolderPath() with the FOLDERID_Profile flag set."_ - I'm not entirely sure what to make of that...